### PR TITLE
countdown-timer-in-C#

### PR DESCRIPTION
**Pull request by rondhi**

### CODE REVIEW

File: extensions/countdown-timer-in-c#.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
...WkhgYVCOtM4sqnhwCep25aPDd233+omKV6854skdatEUlwJtL40+ERqPYvzxfvSFtwn85/nw2AwWyomj+GQGcy2pcGR0CywBGUCOdQRyDaUoRPwS2KWPfhFUc/+KvdNO1803tQXbU4tbpsEYaqDDjSFEQOxkaYR03SkiX1Jv4Uvvq/QT75S6Hc0iPy2bSLp+xTpgkIUUNhGpk6J5cOLiVdfb74+9H/TuAfuLKUlkaJUA+fGFHEqOcJE6Y4jdYdYxhvFPfWu78+//7LFIowSw0A6wBTpUMUg27QwsqCudeCfofRXR4rXY/dFofKNU8xjYBVAeWmGkQISicEpMmzdRDonJnJ8ZSJqOkQSYEGqb5NhHofXr+3ywgFlM5Mh33IsKO+g0nUkFYhR7HOLO4rab1TqVKQ8weTkMv+bk/fs/G8LSBC7fEB4MSJ9jWmasCUShEgwjp/FY+Ugm9nEdpQwNOuLTnp/qOb52oikvmnpGpTclOk2FDkAS0Z1jjRCOWeGrxPff0NEGveIXP2lIGl8KSQFgwlKw4hYhN5bR0gAJ5yEpWVq8MD+00HSZ0LBmTc7+WoASQkZmCtbgfLhOExsnxDzjf6yMIMkwX9JSJJycn0xJonkmqZjDUmHY6TD6Rw5BsPIdwTXhEUc5b+6UH/3mBQOg5MglcjSNHBE5cDpBAp0UDdkCF8o5cs3CpPNQL7rfP3dHzQNRk2iK4U488FzFDWRrTEbUSUFZhjrjs++bbH+tcFrWERpRIdak0ioOi1hIG4qgQi3bQNqbcsy3upPtWfx9m/wfsWTJtEsm3PKik6BjoUOBQHA2NIEhzOUpiz1Dbok3xK8mmGZjoSCnDINzonS0hBzoEh3aHZSNH2scettwHsaz9915N19zeD+yxqrk+xyK1Rr9a8v/EaBzXxuOthBxKfZZYMlETO5QNixufINKC/xe/lGwePbljds8ZpQRjnMUcg3QA06t7Ws1JRIahgqGWZo6sAP/27xfnHwMiHDGtnNjmErHUGeFYgLiGWa4Zu+JJapsz9Zi9ewoLLwbQsJOMYgHcIV4lm/x/YNw8S20Ikm3yZ49dTqvX9N6rtPvlT6BvehcjSdrHuHmQnRQhJERXZKNzUJ73/KNu8BuAqwvPgyUVkCoC8hsTgM9tYhz+gCUrgpqKZzw2DM/H5bvdlbMbPw393g7/8HCINqPX45AAA=`
```
Comment (GoWMan813) [inline extensions/countdown-timer-in-c#.md]: Need Import Code formatted as
image

File: extensions/countdown-timer-in-c#.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Streamer.bot Countdown Timer Import Code
```
Comment (GoWMan813) [inline extensions/countdown-timer-in-c#.md]: Need to add this to the top of the page 
image

Comment (GoWMan813) [inline extensions/countdown-timer-in-c#.md]: Add 
image

Comment (GoWMan813) [reply extensions/countdown-timer-in-c#.md]: Above this line

**Review by GoWMan813 — CHANGES_REQUESTED** [review]

**Comment by rondhi** [comment]
Oops I don't know what I pressed. Lemme see if I can change those.

**Pull request merged**